### DISCUSSION TITLE
Binary variables not working.

### DISCUSSION THREAD

**Discussion by davidcarails** [google/or-tools#3280] [Linear Solver questions] [answered]
I am running this testcase lp_model.mps with the "solve" standalone executable. (compiled from examples/cpp/solve.cc)
The solution values are not binary.
I get values like 0.25 and 0.5

The solve executable build command I used is below
<pre>
make run SOURCE=examples/cpp/solve.cc
</pre>

model file below
[lp_model.mps.gz](https://github.com/google/or-tools/files/8629415/lp_model.mps.gz)

The solving command I used is below
<pre>
gunzip lp_model.mps.gz

solve --input lp_model.mps --sol_file sol.txt
</pre>

**Comment by lperron** [comment, marked as answer]
The default solver is glop, which is a simple (LP) solver, not a MIP solver.

**Reply by davidcarails** [reply]
thanks
